main.tf

```diff
@@ -42,6 +42,7 @@ module "container_definition" {
   container_cpu                = "${var.container_cpu}"
   container_port               = "${var.container_port}"
   healthcheck                  = "${var.healthcheck}"
+  environment                  = "${var.environment}"
```

Comment:
Can you update readme.yaml example with environment variables. Then build readme.